js/scripts.js

```diff
@@ -178,64 +178,68 @@ fetch(LocationApiUrl)
         let urlSupermarket;
         let phoneSupermarket;
         
-        for ( var i = 0; i < data.markets.length; i++) {
-
-            nameSupermarket = data.markets[i].name;
-            addressSupermarket = data.markets[i].contact.address.street;
-            zipCodeSupermarket = data.markets[i].contact.address.city.zipCode;
-            citySupermarket = data.markets[i].contact.address.city.name;
-            urlSupermarket = data.markets[i].url;
-            phoneSupermarket = data.markets[i].contact.phoneNumber;
-
-            // console.log(phoneSupermarket);
-            // console.log(urlSupermarket);
-            // console.log(citySupermarket);
-            // console.log(zipCodeSupermarket);
-            // console.log(addressSupermarket);
-            // console.log(nameSupermarket);
- 
-            storesHTML +=  `
-                <div class="store-container">
-                <div class="store-info-container">
-                    
-                    <div class="store-name font-weight-bold text-large">
-                        <span>${nameSupermarket}</span>
-                    </div>
-                    
-                    <div class="store-address">
-                        <span>${addressSupermarket},</span>
-                        <span>${zipCodeSupermarket} ${citySupermarket}</span>
-                    </div>
-                    
-                    <div class="store-opening-hours">
-                        <span class="font-weight-bold">Opening times:</span> <span> Weekday from 07:00 to 20:00 </span>
-                    </div>
-
-                    <div class="store-phone-number">
-                        ${phoneSupermarket}
-                    </div>
-
-                </div>
-
-                <div class="store-number-container">
-
-                    <div class="website"> 
-                        <span>
-                            <a href="${urlSupermarket}"><i class="fa fa-globe fa-2x"></i></a>
-                        </span>
-                    </div>
-
-                    <div class = "store-number">
-                        
-                        ${i + 1} 
-
-                    </div>
-                    
-                </div>
-                </div>
-                `;
-
-        }
+            function displayStores() {
```

Comment:
You need to write these functions outside then and leave only invocation in here.
In case you need a refresh how to pass a data inside a function here is a reference - https://www.w3schools.com/js/js_function_parameters.asp